test/features/user_records_an_application_test.rb

```diff
@@ -10,6 +10,29 @@ def test_user_can_populate_new_application_form_from_query_param
     assert_equal query_parameter, find_field("URL of the Job Posting").value
   end
 
+  def test_user_redirects_to_confirmation_message_from_bookmarklet_application_form
+    query_parameter = "true"
+    sign_in_to_site
+
+    visit new_application_path(bookmarklet: query_parameter)
+    fill_in_all_application_form_fields
+    select_a_status
+    save_application
+    save_and_open_page
+
+    assert page.has_content? 'Your application has been submitted!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.